### DISCUSSION TITLE
Upgrading self-hosted to version 2.0

### DISCUSSION THREAD

**Discussion by snvgglebear** [Budibase/budibase#8007] [Help]
I am self-hosting budibase with docker-compose. Can I upgrade within the app, or do I need to redownload the docker images?

**Comment by seoulaja** [comment]
Hi @gondilon, yes, I think you just need to hit check for update button 
![image](https://user-images.githubusercontent.com/15101654/192684697-6dd5b062-90a9-4c84-a273-d87f03f04a48.png)

It works fine for me
